recipe/0026-toc-opera/manifest.json

```diff
@@ -0,0 +1,465 @@
+{
+  "@context": [
+    "http://www.w3.org/ns/anno.jsonld",
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
```

Comment:
This should instead be just the presentation 3 context (despite the current template).
See: https://iiif.io/api/presentation/3.0/#46-linked-data-context-and-extensions